lib/utils.js

```diff
@@ -35,6 +36,65 @@ async function getSimpleCypressConfig(config) {
   };
 }
 
+async function convertJsonScreenshotsToBase64(jsonsDir,screenshotsDir) {
+  const dirResult = await fs.readdirSync(jsonsDir);
+  const jsonParsed = [];
+
+  for (let i = 0; i < dirResult.length; i++) {
+    const element = dirResult[i];
+    const filePath = path.resolve(jsonsDir,element);
+    jsonParsed.push({
+      file: filePath, data: await fse.readJson(filePath)
+    });
+  }
+
+  function convertContextImgToBase64(contextObj){
```

Comment:
I prefer that `convertContextImgToBase64` & `genereatebase64Context` wouldn't be an inner functions of `convertJsonScreenshotsToBase64`, you can move them outside of the function block